src/components/Layout/Sidebar.module.scss

```diff
@@ -105,6 +105,14 @@
   display: block;
 }
 
+.item__nonProduct.item__current {
+  background-color: var(--mm-color-sidebar);
+}
+
+.item__nonProduct.item__current > .itemLink {
+  background: var(--mm-color-background) !important;
```

Comment:
You should try to avoid using `!important` as much as possible, as it can make CSS hard to maintain.  Instead, you can take advantage of the cascading nature of styles by moving these styles lower in the file and/or using a more specific selector.

I'd probably move these 2 styles below the last `list__level2` style and above `@include variables.breakpoint('lg')`